src/java/org/apache/cassandra/tools/nodetool/DescribeCluster.java

```diff
@@ -86,19 +84,28 @@ public void execute(NodeProbe probe)
         out.println("\tUnreachable: " + unreachableNodes.size());
 
         Map<String, String> tokensToEndpoints = probe.getTokenToEndpointMap(withPort);
+        StringBuilder errors = new StringBuilder();
         Map<String, Float> ownerships = null;
         try
         {
             ownerships = probe.effectiveOwnershipWithPort(keyspace);
         }
         catch (IllegalStateException ex)
         {
-            ownerships = probe.getOwnershipWithPort();
-            out.println("Error: " + ex.getMessage());
+            try
+            {
+                ownerships = probe.getOwnershipWithPort();
+                errors.append("Note: ").append(ex.getMessage()).append("%n");
+            }
+            catch (RuntimeException e)
```

Comment:
If somehow in the future `probe.getOwnershipWithPort` throws non-RuntimeException, this catch fails to hide the trace. Compiler does not help to check. Maybe we want to cache `Exception` instead? 
It is a tiny tiny concern, but just want to bring it up.